src/client/CodeEditor/json1PresenceBroadcast.js

```diff
@@ -1,4 +1,5 @@
 import { EditorView } from 'codemirror';
+import { username } from '../Settings.tsx';
```

Comment:
This pattern will not work. We need to pull in the username from some state somewhere, then pass that in.